README.md

```
@@ -1,36 +1,88 @@
+#### 2: Run the Spring Boot Application
+- mvn spring-boot:run
 
-Link the PR to the issue you're solving.
+_______
 
-Include a clear description of your solution.
+## REST API Endpoints
+
+### Bank Endpoints (/api/banks)
+
+Method  |	Endpoint    |	Description
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 10

(MD010, no-hard-tabs)